docs/examples/academy/33. Analysing Imports Exports Data.ipynb

```diff
@@ -229,64 +228,62 @@
       "text/plain": [
-       "<Figure size 576x432 with 1 Axes>"
+       "<Figure size 800x600 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "top10 = imports_df.sum().sort_values(ascending=False).head(10)\n",
+    "top10 = imports_df.sum(numeric_only=True).sort_values(ascending=False).head(10)\n",
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.